reports/report.md

```diff
@@ -1222,6 +1224,23 @@ Description of the high issue.
 
 
 
+## H-19: Sending native Eth is not protected from these functions.
+
+Description of the high issue.
+
+<details><summary>1 Found Instances</summary>
+
+
```

Comment:
Only one instance found? Should we expect more?

Reply:
No, just one I think.